.github/workflows/e2e.yml

```diff
@@ -9,17 +9,17 @@ jobs:
     timeout-minutes: 60
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
           node-version: '14.x'
       - name: Install dependencies
-        run: npm ci
+        run: npm install
```

Comment:
Any particular reason for this change?
npm ci will run a clean install and it's meant for CI/CD pipelines : reference [npm ci docs](https://docs.npmjs.com/cli/v8/commands/npm-ci)

Reply:
Nice catch, forgot that there while debugging. I have removed it.